### ISSUE TITLE
App summary repeats name

### ISSUE THREAD

**Issue by bragefuglseth** [naoxio/inner_breeze#1]
The app summary currently reads like this:

> InnerBreeze: A Breathing Meditation App.

This summary is almost always displayed next to / close to the app name, which makes the name in the summary unnecessary. I suggest removing the app name from it, dropping the punctuation mark at the end and changing the summary to sentence case for consistency with other apps:

> A breathing meditation app

Or, even better, reword it to start with an imperative verb:

> Meditate through breathing

If you're interested, you can read Flathub's [guidelines for app summaries](https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines#summary).

**Comment by waotzi** [comment]
Thank you, I have updated the metainfo, it will be there on the next release :+1:
